src/components/modals/login-modal/LoginContent.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import './login-content.less';
+import Text from 'antd/es/typography/Text';
+
+interface Props {
+  switchToForgotPasswordPage: () => void;
+}
+
+const LoginContent: (props: Props) => JSX.Element = (props: Props) => {
+  const forgotPasswordLink = (
+    <Button
+      onClick={props.switchToForgotPasswordPage}
+      className="page-link"
+      type="link"
+    >
+      here
+    </Button>
+  );
```

Comment:
```suggestion
const LoginContent<Props>: (props: Props) => JSX.Element = (props: Props) => {
  const history = useHistory();
  const handleClickForgotPassword = (): void => {
    history.push('forgot password page link')
  } 
  const forgotPasswordLink = (
    <Button
      onClick={handleClickForgotPassword}
      className="page-link"
      type="link"
    >
      here
    </Button>
  );
```
Will this function always point to the same page?